kobo/apps/accounts/mfa/tests/test_api.py

```diff
@@ -45,8 +47,12 @@ def test_user_methods_with_date(self):
         for field in expected_fields:
             self.assertTrue(field in results[0])
 
+    @override_config(MFA_ENABLED=True)
     def test_mfa_activation_always_creates_new_secret(self):
         self.client.login(username='anotheruser', password='anotheruser')
+        anotheruser = User.objects.get(username='anotheruser')
+
+        mfa_availability = MfaAvailableToUser.objects.create(user=anotheruser)
```

Comment:
As per discussion, I think this is not needed if `MfaAvailableToUser` list is empty. 
You can revert this change if it is case.

Reply:
Right, I removed it